product_docs/docs/pgd/5.6/reference/functions.mdx

```diff
@@ -290,9 +292,38 @@ bdr.wait_slot_confirm_lsn(slot_name text DEFAULT NULL, target_lsn pg_lsn DEFAULT
 
 #### Parameters
 
--   `slot_name` &mdash; Name of replication slot or, if NULL, all PGD slots (only).
--   `target_lsn` &mdash; LSN to wait for or, if NULL, use the current write LSN on the
-    local node.
+| Parameter    | Description                                                                 |
+|--------------|-----------------------------------------------------------------------------|
+| `slot_name`  | Name of the replication slot to wait for. If NULL, waits for all PGD slots. |
+| `target_lsn` | LSN to wait for. If NULL, uses the current write LSN on the local node.     |
+
+### `bdr.wait_node_confirm_lsn`
+
+Wait until a node passes a certain LSN.
+
+This function allows you to wait until the last write on this session was replayed to one or all nodes.
+
+Upon being called, the function waits for a node to pass a certain LSN.
+If no LSN is supplied, the current wal_flush_lsn (using the `pg_current_wal_flush_lsn()` function) position is used on the local node.
+Supplying a node name parameter tells the function to wait for that node to pass the LSN. 
+If no node name is supplied (by passing NULL), the function waits until all the nodes pass the LSN,
+
+The function operates by polling every 1000 ms for changes from other nodes. Those changes are detected either by looking up the `confirmed_flush_lsn` from the node's replication slot or by connecting to the node directly and querying the results of `pg_replication_origin_progress()`, passing the local node as origin.
+
```

Comment:
Add here following details:

If using slot to confirm and the slot is dropped concurrently, the wait ends for that slot. If a node is currently down and isn't updating its slot or cannot connect to node, then the wait continues. You might want to set statement_timeout to complete earlier in that case.